app/posts/manage-teacher-training-applications/2021-03-18-self-service-registration.md

```diff
@@ -0,0 +1,436 @@
+---
+title: Self-service registration for higher education institutions and their school direct partners
+description: Exploring ways to help higher education institutions register and invite their school direct partners to use the Manage service
+date: 2021-03-18
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+How might we provide a self-service journey for higher education institutions (HEIs) and their school direct partners (SDs) to register on the Manage service?
+
+## Why we are doing this
+
+Around 70 HEIs have not yet registered to use Manage. On average each of them accredits courses for 10 SDs, many of which are also not registered on Manage. We do not have contact details for all the SDs.
+
+We need to make sure that all these organisations have registered to use Manage before the UCAS service is turned off in October 2021.
+
+Adding them all manually will be a great deal of work for the transition team. We want to make the process more efficient, while not placing a significant burden on HEIs or SDs.
+
+## Requirements
+
+Before we started designing possible solutions, we spoke to members of the provendor and transition teams about their requirements.
+
+These requirements are not presented as user needs, since this work is being done primarily for the benefit of teams within DfE.
```

Comment:
I wonder if these needs pointing out?

Reply:
I think it's important to note where the requirements came from, i.e., not from a validated user need.

What do you think @johndoates?